src/core/layout/navbar/responsive-drawer/desktop-navbar.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+import { Hidden } from '@material-ui/core'
+
+import { Login } from 'login'
+import { AppBar } from 'core/layout/navbar/appbar'
+import { Header } from 'core/layout/navbar/header'
+
+export const DesktopNavbar = () => <Hidden xsDown>
```

Comment:
Since we're already inside the `<Navbar/>` component, let's remove the navbar suffix here.

Reply:
Done